_data/nav.yml

```diff
@@ -1,96 +1,91 @@
-          - text: 96Boards
-            url: /services/96boards/
-          - text: Toolchain Optimization
-            url: /services/toolchain-optimization-services/
-          - text: Open Source Consultancy
-            url: /services/open-source-consultancy/
-          - text: Testing and Validation
-            url: /services/testing-validation-services/
-          - text: Hands on Training
-            url: /services/hands-on-training/
-          - text: Downloads
-            url: /downloads/
-    - title: Support
-      url: /support/
+  - options:
+      - text: Overview
+        url: /about/
+      - options:
+          - text: The Board
+            url: /about/team/board/
+          - text: Executive Team
+            url: /about/team/executive/
+          - text: TSC
+            url: /about/team/technical-steering-committee/
+          - text: Awards
+            url: /awards/
+        text: Team
+      - options:
+          - text: News
+            url: /news/
+          - text: Blog
+            url: /blog/
+          - text: Events
+            url: /events/
+          - text: Planet Linaro
+            url: /planet/
+        text: News & Events
+      - text: Careers
+        url: /careers/
+      - text: Contact Us
+        url: /contact/
+    title: About
+  - options:
+      - text: Overview
+        url: /engineering/
+      - text: Artificial Intelligence
+        url: /engineering/artificial-intelligence/
+      - text: Autonomous Vehicles
+        url: /engineering/autonomous-vehicles/
+      - text: Consumer
+        url: /engineering/consumer/
+      - text: Core Engineering on Arm
+        url: /engineering/core/
+      - text: Datacenter & Cloud
+        url: /engineering/datacenter-and-cloud/
+      - text: Edge & Fog Computing
+        url: /engineering/edge-and-fog-computing/
+      - text: High Performance Computing
+        url: /engineering/high-performance-computing/
+      - text: IoT & Embedded
+        url: /engineering/iot-and-embedded/
+    title: Engineering
+  - title: Membership
+    url: /membership/
+  - options:
+      - text: Overview
+        url: /services/
+      - text: Bootloaders
+        url: /services/bootloaders/
+      - text: 'BSP’s, Builds and Support'
+        url: /services/bsp-builds-support/
+      - text: Hands on Training
+        url: /services/hands-on-training/
+      - text: Kernels and Long Term Support
+        url: /services/kernel-lts/
+      - text: Open Source Consultancy
+        url: /services/open-source-consultancy/
+      - text: Power Management Tuning
+        url: /services/power-management/
+      - text: Security
+        url: /services/security/
+      - text: Testing and Validation
+        url: /services/testing-validation-services/
+      - text: Toolchain Optimization
+        url: /services/toolchain-optimization-services/
+      - text: 96Boards
+        url: /services/96boards/
+    title: Services
+  - options:
```

Comment:
We need to add an "Overview" page or something similar to link to the /support/ page as the main top level nav items with a dropdown aren't linked as they toggle the dropdown :)